RecoPixelVertexing/PixelTriplets/plugins/GPUCACell.h

```diff
@@ -306,13 +307,20 @@ class GPUCACell {
             ((!startAt0) && hole0(hh, cells[tmpNtuplet[0]])))
 #endif
         {
```

Comment:
maybe, add a comment stating that `8` is the maximum number of hits expected to have in a track (or a more accurate description)